tests/api_connexion/endpoints/test_dag_source_endpoint.py

```diff
@@ -40,7 +49,43 @@ def setUpClass(cls) -> None:
     def setUp(self) -> None:
         self.client = self.app.test_client()  # type:ignore
 
-    @pytest.mark.skip(reason="Not implemented yet")
-    def test_should_response_200(self):
-        response = self.client.get("/api/v1/health")
-        assert response.status_code == 200
+    @parameterized.expand([("True",), ("False",)])
+    def test_should_response_200_text(self, store_serialized_dags):
+        serializer = URLSafeSerializer(conf.get('webserver', 'SECRET_KEY'))
+        with conf_vars(
+            {("core", "store_serialized_dags"): store_serialized_dags}
+        ):
+            dagbag = DagBag(dag_folder=EXAMPLE_DAG_FILE)
+            dagbag.sync_to_db()
+            first_dag: DAG = next(iter(dagbag.dags.values()))
+
+            url = f"/api/v1/dagSources/{serializer.dumps(first_dag.fileloc)}"
+            response = self.client.get(url, headers={
+                "Accept": "text/plain"
+            })
+
+            self.assertEqual(200, response.status_code)
+            self.assertIn("Example DAG demonstrating the usage of the BashOperator.", response.data.decode())
+            self.assertEqual('text/plain', response.headers['Content-Type'])
+
+    @parameterized.expand([("True",), ("False",)])
+    def test_should_response_200_json(self, store_serialized_dags):
```

Comment:
Can you test 404?

Reply:
Shouldn't the test parameter be `store_dag_code` instead of `store_serialized_dags`?

Reply:
Tests added